INSTALL.md

```diff
@@ -81,6 +81,14 @@ Ginkgo adds the following additional switches to control what is being built:
     this option see the
     [`ARCHITECTURES` specification list](https://github.com/ginkgo-project/CudaArchitectureSelector/blob/master/CudaArchitectureSelector.cmake#L58)
     section in the documentation of the CudaArchitectureSelector CMake module.
+* `-DGINKGO_WINDOWS_SHARED_LIBRARY_RELPATH=<path>` where <path> is a relative
+    path with `PROJECT_BINARY_DIR`. Users must to add the absoulte path
+    (`PROJECT_BINARY_DIR`/`GINKGO_WINDOWS_SHARED_LIBRARY_RELPATH`) into the
+    environment variable PATH when building shared libraries and executable
+    program, default is `windows_shared_library`.
+* `-DGINKGO_CHECK_PATH={ON, OFF}` checks the environment variable PATH is valid.
+    It is valid only when building shared libraries and executable program,
+    default is `ON`
```

Comment:
Here you mean that this is done only when building shared libraries (static does not need the `check` ?).

Reply:
No, static does not need the `check`.
Because Windows does not have RPATH, I need to put all shared libraries into each program's folder or add their path into the environment `PATH` when building shared libraries.
For static libraries, cmake link them with absolute path, so it has no problem.

Reply:
OK, thanks for the confirmation.

In that case, rather than `It is valid only...` , maybe change it to `It is checked only...`. Since the previous line also has `valid` but for another meaning, changing the world may be helpful.